#!/usr/bin/env python

##**************************************************************************
 #                           DynParameters_2Entries.cfg                    *
 #                           -------------------                           *
 # copyright            : (C) 2019 by Ignacio Abadia                       *
 # email                : iabadia@ugr.es                                   *
 #**************************************************************************/

##**************************************************************************
 #                                                                         *
 #   This program is free software; you can redistribute it and/or modify  *
 #   it under the terms of the GNU General Public License as published by  *
 #   the Free Software Foundation; either version 3 of the License, or     *
 #   (at your option) any later version.                                   *
 #                                                                         *
 #**************************************************************************/

PACKAGE = "edlut_ros"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("Time_Window", double_t, 0, "Time Window",    1.0, 0)
gen.add("First_Min_Neuron",    int_t,    0, "First Group Min Neuron", 0, 0)
gen.add("First_Max_Neuron",    int_t,    0, "First Group Max Neuron", 0, 0)
gen.add("Second_Min_Neuron",    int_t,    0, "Second Group Min Neuron", 0, 0)
gen.add("Second_Max_Neuron",    int_t,    0, "Second Group Max Neuron", 0, 0)
gen.add("Stop",   bool_t,   0, "Stop plot",  False)
gen.add("Display", bool_t, 0, "Display graph", False)


exit(gen.generate(PACKAGE, "edlut_ros", "DynParameters_2Entries"))
